.travis.yml

```diff
@@ -1,13 +1,14 @@
-sudo: false
```

Comment:
The `sudo:` key has been deprecated, see https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure